packages/terra-form-radio/src/Radio.module.scss

```diff
@@ -144,8 +144,8 @@
       }
 
       .native-input:checked + .outer-ring {
-        border-color: var(--terra-form-radio-outer-ring-checked-hover-border-color, transparent);
-        border-width: var(--terra-form-radio-outer-ring-checked-hover-border-width, 0);
+        border-color: var(--terra-form-radio-outer-ring-hover-checked-border-color, transparent);
+        border-width: var(--terra-form-radio-outer-ring-hover-checked-border-width, 0);
```

Comment:
I'd update this to `--terra-form-radio-hover-checked-outer-ring-border-width` to match how we are doing the variable on line 143 if the linter allows it.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/41ede6d5aee20c73d2c805f199e66417fbe049e9#diff-e32405d004b4f846ad58be3415e40273R147

Reply:
Resolved in this commit: https://github.com/cerner/terra-core/pull/2125/commits/41ede6d5aee20c73d2c805f199e66417fbe049e9